RecoParticleFlow/PFClusterProducer/plugins/CorrectedECALPFClusterProducer.cc

```diff
@@ -131,6 +131,7 @@ void CorrectedECALPFClusterProducer::fillDescriptions(edm::ConfigurationDescript
     psd0.add<bool>("applyCrackCorrections", false);
     psd0.add<bool>("applyMVACorrections", false);
     psd0.add<bool>("srfAwareCorrection", false);
+    psd0.add<bool>("srfAwareCorrection",false);
```

Comment:
This was added twice, instead of "setEnergyUncertainty"

Reply:
urgh, yes this came from the fact I had to rebase back to 11_X due to code formats :(

Reply:
thanks fixed now